packages/turbo-repository/rust/src/lib.rs

```diff
@@ -131,4 +135,58 @@ impl Workspace {
 
         Ok(map)
     }
+
+    /// Given a set of "changed" files, returns a set of packages that are
```

Comment:
Should probably document that the changed files are expected to:
 - be relative to workspace root
 - use current system path separator